src/host/screenInfo.cpp

```diff
@@ -1422,7 +1421,7 @@ bool SCREEN_INFORMATION::IsMaximizedY() const
     try
     {
         newTextBuffer = std::make_unique<TextBuffer>(coordNewScreenSize,
-                                                     gci.GetDefaultAttributes(),
+                                                     TextAttribute{},

```

Comment:
**whoa**. this one scares me!! the session default attributes are more important in conhost than terminal. why the change?

Reply:
oh but it defaults to `{Default, Default}` doesn't it

Reply:
Oh I thought this was the way to do it post-#6506

Reply:
ohhhhhhhhh okay